ludwig/features/base_feature.py

```diff
@@ -209,13 +209,14 @@ def train_loss(self, targets: Tensor, predictions: Dict[str, Tensor]):
     def eval_loss(self, targets: Tensor, predictions: Dict[str, Tensor]):
         loss_class = type(self.train_loss_function)
         prediction_key = LOSS_INPUTS_REGISTRY[loss_class]
-        return self.eval_loss_function(predictions[prediction_key], targets)
+        return self.eval_loss_function(predictions[prediction_key].detach(),
```

Comment:
I'm wondering if `.clone().detach()` would make a difference regarding the issue where the reported combined loss isn't the same as the reported loss for the output feature.

Reply:
@justinxzhao   thank you for the suggestion.  Tried `clone().detach()`.  No change in output
```
╒═════════╤════════╤════════════╤═════════════╕
│ label   │   loss │   accuracy │   hits_at_k │
╞═════════╪════════╪════════════╪═════════════╡
│ train   │ 0.1200 │     0.9786 │      0.9981 │
├─────────┼────────┼────────────┼─────────────┤
│ test    │ 0.1173 │     0.9793 │      0.9985 │
╘═════════╧════════╧════════════╧═════════════╛
╒════════════╤════════╕
│ combined   │   loss │
╞════════════╪════════╡
│ train      │ 0.0713 │
├────────────┼────────┤
│ test       │ 0.0636 │
╘════════════╧════════╛
```
As I recall the fix in TF version, it required rationalizing how the loss was captured between two different ways.  I'll have to go back the the original fix to recall what was done.

For the moment, I'm leaving the code as-is.

Reply:
Thanks for checking on this! Perhaps it's a good thing that we're seeing the same behavior as the TF version.